src/styles/devices.styl

```diff
@@ -0,0 +1,15 @@
+@import 'cozy-ui'
+
+.devices-view
+    h2
+        horizontally-padded 2em
+
+    .coz-btn--revoke
+        color dodger-blue
```

Comment:
alignment and order are wrong here.
from the outside to the inside which gives paddings > bg > color